decisions/0006-program-slugs-in-publisher.rst

```diff
@@ -0,0 +1,30 @@
+6. Program Slugs in Publisher
+-------------------------------
+
+Status
+------
+
+Accepted (July 2023)
+
+Context
+-------
+
+Ability to update the slugs for programs is needed to be accessible to a group of users.
+These slugs are used in generating marketing url for each type of programs.
+These program slugs will be programmatically generated, just like courses.
+In order to grant the ability to update slugs for programs, we will going to display programs on Publisher due to it's ease of access control.
+Instead of displaying all program information on the publisher, we will only show the editable slug field.
+
+Decision
+--------
+
+A new tab will be added on Publisher named "Programs/Degrees" in the header.
+Upon clicking the tab, the user will be shown a list of programs with minimal info.
+Each item in the list will take the user to it's detail page where an input field will be available for slugs and a save button.
+To access the program list and detail pages, every user must be granted the necessary permissions.
+
+
+Consequences
```

Comment:
Probably, we can add 1-2 lines on how it will impact existing publisher users.

Reply:
Is this going to affect the existing users given they don't have permissions? They won't be able to interact if they don't have access.